### ISSUE TITLE
Tests fail on Windows, also on Mac when cross-compiling to windows

### ISSUE THREAD

**Issue by dmagyar** [google/gopacket#356]
Hi!

I'm trying to get gopacket work on Windows and Mac and it seems current git version fails to test:

```
C:\Users\xxx\go\src\github.com\google\gopacket\pcap>go test
# github.com/google/gopacket/pcap
.\pcap_windows.go:14:10: undefined: Handle
FAIL    github.com/google/gopacket/pcap [build failed]

C:\Users\xxx\go\src\github.com\google\gopacket\pcap>
```

Of course my Go environment works, I can compile code happily on both platforms.
When tried from Mac:

```
DMsMAC:pcap dmagyar$ GOOS=windows GOARCH=386 go test
# github.com/google/gopacket/pcap
./pcap_windows.go:10:10: undefined: Handle
FAIL	github.com/google/gopacket/pcap [build failed]
DMsMAC:pcap dmagyar$ 
```

Any pointers?

**Comment by gconnell** [comment]
I'd love to help, but lacking a personal windows machine I tend to rely on others to debug Win build issues.

Could you try adding this to the top of the pcap_windows.go file and see if it helps?

```go
// +build windows
```

**Comment by dmagyar** [comment]
Thanks, I tried - not helping :(  I also tried a previous "release" tgz and it did not work either

**Comment by notti** [comment]
This should be fixed by #568
